src/Files.App/ValueConverters/ColorToSolidColorBrushValueConverter.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.UI.Xaml.Data;
+using Microsoft.UI.Xaml.Media;
+using System;
+using Windows.UI;
+
+namespace Files.App.ValueConverters
+{
+	public class ColorToSolidColorBrushValueConverter : IValueConverter
+	{
+		public object? Convert(object value, Type targetType, object parameter, string language)
+		{
+			if (null == value)
+				return null;
+
+			if (value is Color)
+			{
+				Color color = (Color)value;
+				return new SolidColorBrush(color);
```

Comment:
we'd probably want to convert the alpha channel to `255` to preserve behavior? 
or is it intentional that we want it to look opaque?

```cs
return new SolidColorBrush(Color.FromArgb(255, color.R, color.G, color.B));
```

Reply:
This is intentional, it's designed this way so that the app still shows the mica background.